### PR TITLE
Add pull request and issue templates

### PR DESCRIPTION
**Pull request by robobario**
Inspired (copy + paste) by the Strimzi project templates, thanks!

### CODE REVIEW

File: ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+_Please use this to only for bug reports_. For questions or when you need help, you can use the [GitHub Discussions](https://github.com/kroxylicious/kroxylicious/discussions)
```
Comment (robobario) [inline ISSUE_TEMPLATE/bug_report.md:10]: I don't think there's a way to inject a different link per repo, maybe you could use a relative link like `../discussions`? Or we could turn on discussions at the organisation level and use https://github.com/kroxylicious/discussions

File: ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions and Discussions
+    url: https://github.com/kroxylicious/kroxylicious/discussions
```
Comment (robobario) [inline ISSUE_TEMPLATE/config.yml:4]: as above, not sure if we should point all the kroxy repositories here

File: ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (k-wall) [reply ISSUE_TEMPLATE/bug_report.md:10]: +1

Comment (SamBarker) [reply ISSUE_TEMPLATE/bug_report.md:10]: I'm on the fence as to org wide -vs- per project discussions.

**Comment by robobario** [comment]
Will go with per-repo templates for now since we may not want them all to point at the same discussion/support link.

**Pull request closed without merging**